e2e/cypress/integration/dashboard-transloadit.spec.ts

```diff
@@ -470,9 +344,12 @@ describe('Dashboard with Transloadit', () => {
     cy.wait(['@assemblies', '@resumable'])
 
     cy.get('.uppy-StatusBar-statusPrimary').should('contain', 'Complete')
-  })
+  }).timeout(40000)
```

Comment:
nit: for readability
```suggestion
  }).timeout(40_000)
```